web/wp-content/themes/bumblebee/header.php

```diff
@@ -10,10 +10,14 @@
  */
 
 wp_enqueue_style( 'bumblebee-style-header', get_stylesheet_directory_uri() . '/header.css', [], '1.0.2' );
+$fbid = get_theme_mod( 'fbid', false );
```

Comment:
See https://readersdigest.atlassian.net/browse/DRDT-99 to add this field to the customizer (so that POs can change the value without WP CLI)